scripts/lib/mkcloud-common.sh

```diff
@@ -533,6 +531,13 @@ function dist_to_image_name
     echo "$image.qcow2"
 }
 
+: ${zypper:=run_zypper}
+function run_zypper
+{
+    local params=${zypper_override_params:---non-interactive --gpg-auto-import-keys --no-gpg-checks}
+    $sudo zypper $params "$@"
+}
```

Comment:
I think this does not work when $zypper is passed to e.g. ssh command (see `onadmin_allow_vendor_change_at_nodes`)